src/Tizen.NUI/src/public/Visuals/VisualObject/VisualBase.cs

```diff
@@ -358,20 +366,32 @@ public float Opacity
         /// The default value is VisualFittingModeType.DontCare.
         /// If user set one of Transform property, it will be set as VisualFittingModeType.DontCare automatically.
         /// </remarks>
+        /// <remarks>
+        /// Fitting mode is only avaliable when the visual has original size.
+        /// For example, ImageVisual and TextVisual support FittingMode, but ColorVisual and BorderVisual doesn't support.
```

Comment:
```suggestion
        /// For example, ImageVisual and TextVisual support FittingMode, but ColorVisual and BorderVisual don't support.
```

Reply:
Thank you for your comment! don't is more right word than doesn't. Fix done.